functions/Remove-SqlOrphanUser.ps1

```diff
@@ -52,201 +55,314 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 https://dbatools.io/Remove-SqlOrphanUser
 
 .EXAMPLE
-Remove-SqlOrphanUser -SqlServer sql2005 
+Remove-SqlOrphanUser -SqlInstance sql2005
```

Comment:
Please continue to use -SqlServer in examples until we move to 1.0. We have a lot of screenshots on the website that I will not have time to update until there's a dedicated time to do so.